src/main/java/com/github/technus/tectech/thing/metaTileEntity/hatch/GT_MetaTileEntity_Hatch_InputData_Wireless.java

```diff
@@ -0,0 +1,212 @@
+package com.github.technus.tectech.thing.metaTileEntity.hatch;
+
+import static com.github.technus.tectech.thing.metaTileEntity.hatch.GT_MetaTileEntity_Hatch_DataConnector.EM_D_SIDES;
+import static com.github.technus.tectech.util.CommonValues.MOVE_AT;
+import static gregtech.api.enums.Dyes.MACHINE_METAL;
+import static net.minecraft.util.StatCollector.translateToLocal;
+import static net.minecraft.util.StatCollector.translateToLocalFormatted;
+
+import net.minecraft.client.renderer.texture.IIconRegister;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.EnumChatFormatting;
+import net.minecraftforge.common.util.ForgeDirection;
+
+import org.apache.commons.lang3.reflect.FieldUtils;
+
+import com.github.technus.tectech.mechanics.dataTransport.WirelessDataNetwork;
+import com.github.technus.tectech.mechanics.dataTransport.WirelessInventoryDataPacket;
+import com.github.technus.tectech.util.CommonValues;
+import com.github.technus.tectech.util.TT_Utility;
+
+import cpw.mods.fml.relauncher.Side;
+import cpw.mods.fml.relauncher.SideOnly;
+import gregtech.api.enums.Dyes;
+import gregtech.api.enums.Textures;
+import gregtech.api.interfaces.ITexture;
+import gregtech.api.interfaces.tileentity.IGregTechTileEntity;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.implementations.GT_MetaTileEntity_Hatch_DataAccess;
+import gregtech.api.objects.GT_RenderedTexture;
+
+public class GT_MetaTileEntity_Hatch_InputData_Wireless extends GT_MetaTileEntity_Hatch_DataAccess {
+
+    private String owner_uuid;
+    private String owner_name;
+    public static Textures.BlockIcons.CustomIcon DATA_WIRELESS_SLAVE;
+
+    @Override
+    @SideOnly(Side.CLIENT)
+    public void registerIcons(IIconRegister aBlockIconRegister) {
+        super.registerIcons(aBlockIconRegister);
+        DATA_WIRELESS_SLAVE = new Textures.BlockIcons.CustomIcon("iconsets/OVERLAY_DATA_WIRELESS_SLAVE");
+    }
+
+    private ItemStack[] stacks;
+    private WirelessInventoryDataPacket inventoryDataPacket;
```

Comment:
Why not use `mInventory` from the super class and copy the packet to there when it's received.